verif/sim/setup-env.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 # Set verilator and spike related variables
 if [ -z "$VERILATOR_INSTALL_DIR" ]; then
-    export VERILATOR_INSTALL_DIR="$ROOT_PROJECT"/tools/$(ls $ROOT_PROJECT/tools | grep verilator | head -n 1)
+    export VERILATOR_INSTALL_DIR="$ROOT_PROJECT"/tools/verilator-v5.018/
```

Comment:
@valentinThomazic do you see it right?

Reply:
His PR is waiting for this one to be merged, do you need theses changes to make the Github Actions work?

Reply:
I think that setting `VERILATOR_BUILD_DIR` and `VERILATOR_INSTALL_DIR` to something like `tools/verilator` would make it easier since you would not have to care about the current used version of verilator. The install scripts will just install it there.

Reply:
About your last commit, I'm not sure we understood each other !

I meant to say that if you set then env variable `VERILATOR_INSTALL-DIR` (and only this one, my previous comment was not completely accurate) in your CI, then I think you could leave `verif/sim/setup-env.sh` and `verif/regress/install-verilator.sh` as they are. 
Verilator will be installed in the specified path by the script. 

By doing so, having several verilator versions on `tools/` will still be a feature available to the users !

Reply:
I reverted the changes to the `verif/sim/setup-env.sh` and `verif/regress/install-verilator.sh`. If you can double check now the changes :+1: